charts/plugin-site/templates/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "plugin-site.fullname" . }}
+  labels:
+{{ include "plugin-site.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "plugin-site.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
```

Comment:
@timja are you ok to add the maintainer label referencing your Github account?
This will be usefull when/if we create alert for specific services